src/components/stars/starsPage/addStar.tsx

```diff
@@ -195,10 +216,9 @@ const AddStar = ({
           <Select
             variant="standard"
             input={<Input />}
-            defaultValue={currPlatform}
-            {...register('platform')}
+            value={currPlatform}
+            disabled={!setCurrPlatform}
```

Comment:
Not `currPlatform`?

Reply:
If there is no setCurrPlatform in the props (so it's undefined) I disable the field